bin/i18n/resources/pegasus/emails/sync_in.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+
+require_relative '../../../i18n_script_utils'
+require_relative '../../../utils/pegasus_markdown'
+require_relative '../emails'
+
+module I18n
+  module Resources
+    module Pegasus
+      module Emails
+        class SyncIn
+          LOCALIZABLE_FILE_SUBPATHS = %w[
+            hoc_signup_2023_receipt_en.md
+          ].freeze
+
+          def self.perform
+            new.execute
+          end
+
+          def execute
+            progress_bar.start
+
+            I18nScriptUtils.process_in_threads(LOCALIZABLE_FILE_SUBPATHS) do |file_subpath|
+              origin_file_path = File.join(ORIGIN_DIR_PATH, file_subpath)
+              next unless File.exist?(origin_file_path)
+
+              i18n_source_file_path = File.join(I18N_SOURCE_DIR_PATH, file_subpath)
+              i18n_source_file_path = i18n_source_file_path.sub('public/public/', '')
```

Comment:
As I mentioned, you don't need to add these files to any additional directory. Going directly into emails would be good.

Reply:
Oh does the `I18nScriptUtils.copy_file` call here copy the files to another location? I thought this was where the files are placed into our repo?